src/browser/tab/mod.rs

```diff
@@ -58,6 +58,12 @@ pub type ResponseHandler = Box<
     + Sync,
 >;
 
+pub struct SyncSendEvent(pub Arc<Tab>, pub Box<dyn Fn(&Event, &Tab)>);
```

Comment:
is this so that the event listener can have a reference to the `Tab`? why can't you just clone the tab and `move` it into the event listener closure?

Reply:
yeah i previously tried to move the tab into the closure. but that wasnt working, so i changed the SyncSendEvent to accept the tab struct

Reply:
the reason i'm pushing back is that it'll break code that uses the current API, and I think you can get it working with moving the tab (maybe you need to make a new clone and move that?)

Reply:
Yeah i understand, this is my first time contributing, so im a bit inexperienced, i will try doing that. i need some guidance 😅

Reply:
no worries, thanks for taking the time to respond to my feedback 🙂

Reply:
> no worries, thanks for taking the time to respond to my feedback

let me try again with cloning the tab and moving it in the closure.

Reply:
I tried again with cloning the tab and it was working, previously i didnt clone it properly.